clap-v3-utils/src/keygen/mnemonic.rs

```diff
@@ -35,6 +35,21 @@ pub fn word_count_arg<'a>() -> Arg<'a> {
         .help(WORD_COUNT_ARG.help)
 }
 
+pub fn acquire_word_count(matches: &ArgMatches) -> usize {
+    match matches
+        .get_one::<String>(WORD_COUNT_ARG.name)
+        .unwrap()
+        .as_str()
+    {
+        "12" => 12,
+        "15" => 15,
+        "18" => 18,
+        "21" => 21,
+        "24" => 24,
+        _ => unreachable!(),
```

Comment:
This is brittle, if the possible values in `word_count_arg()` are ever updated.
Can you add some comments so it's clear the lists must be kept in sync?

Reply:
Yes, good point. I added the comment.